libs/staging-provider/src/WorkspaceProviderFactory/Credentials/ABSWorkspaceCredentials.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StagingProvider\WorkspaceProviderFactory\Credentials;
+
+use Keboola\StagingProvider\Exception\StagingProviderException;
+use Throwable;
+
+class ABSWorkspaceCredentials implements CredentialsInterface
+{
+    private function __construct(private readonly string $connectionString)
+    {
+    }
+
+    public static function fromPasswordResetArray(array $credentials): self
+    {
+        // This reads the response from the password reset endpoint for ABS backend
+        // https://keboola.docs.apiary.io/#reference/workspaces/password-reset/password-reset
+        try {
+            return new self(
+                $credentials['connectionString'],
+            );
+        } catch (Throwable $e) {
+            throw new StagingProviderException(
+                sprintf(
+                    'Invalid password reset response for ABS backend "%s" - keys: %s',
+                    $e->getMessage(),
```

Comment:
Tohle předávání message z Throwable nás jednou imho vytrestá. Teď to ničemu nevadí.

Reply:
No univerzálně by se neměl catchovat Throwable, ale tady by to ničemu nemělo vadit